modules/manage/partials/kubernetes/tls-update-note.adoc

```diff
@@ -0,0 +1,6 @@
+Enabling or disabling TLS for the RPC listener requires you to delete all Pods that run Redpanda. When you change the `rpc.tls.enabled` setting, or if it is not overridden and you change the global `tls.enabled` option, Redpanda does not automatically apply the change. To apply the change, you must delete all the Pods that run Redpanda.
```

Comment:
I think I'd clarify this a bit more as "not automatically apply" isn't technically correct. How do you feel about:

Redpanda cannot safely apply this change as a RPC listener configurations must be homogenous. To fully rollout the change, all Redpanda Pods must be deleted at the same time such that they all start with the new, updated, RPC listener. This will result in temporary downtime of the cluster. `--force` may be used to speed up the rollout but may result in data loss as redpanda will not be given the opportunity to shutdown gracefully.